src/unix/fs.c

```diff
@@ -49,6 +49,7 @@
     defined(__FreeBSD__)    ||                                            \
     defined(__OpenBSD__)    ||                                            \
     defined(__NetBSD__)
+# include <sys/syslimits.h> /* PATH_MAX, IOV_MAX */
```

Comment:
I've pushed an update that explicitly includes limits.h with the other headers. Seems like a good defensive mood